tagilmo/VereyaPython/timestamped_video_frame.py

```diff
@@ -74,13 +76,16 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.zPos = loadedjson['z']
         self.yaw = loadedjson['yaw']
         self.pitch = loadedjson['pitch']
-        self.modelViewMatrix = np.reshape(np.asarray(loadedjson['modelViewMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
+        self.iheight = loadedjson['img_height'][0]
```

Comment:
I guess it should be the same as pitch, yaw pos
```suggestion
        self.iheight = loadedjson['img_height']
```

Reply:
Ok, but don't remove [0] please.

Reply:
@noskill  Done